docs/tutorials/detecting-a-double-tap.md

```diff
@@ -0,0 +1,10 @@
+---
+title: 'Detecting a double tap'
+tags: [touch, input, tutorial]
+thumb: https://s3-eu-west-1.amazonaws.com/images.playcanvas.com/projects/12/436531/38B52B-image-75.jpg
+---
+
+Sample showing how to detect a double tap on a touch screen.
```

Comment:
An another. I won't spam any more of them.... 😉

Reply:
Ah, I thought I had them all. Should be sorted now